torch/csrc/jit/source_range.h

```diff
@@ -9,25 +9,65 @@
 namespace torch {
 namespace jit {
 
+struct SourceRange;
+using SourceRangeRecord = std::tuple<size_t, std::shared_ptr<SourceRange>>;
+using SourceRangeRecords = std::vector<SourceRangeRecord>;
+
+// Class that keeps track of a serialized debug info table and lazily
```

Comment:
This comment is not useful. I don't know what is in data using this comment.

Reply:
This is implementation detail and doesn't need to be in source_range.h, which is widely included. It can be opaque.